domain_config_ui/domain_config_ui.services.yml

```diff
@@ -0,0 +1,16 @@
+services:
+  domain_config_ui.manager:
+    class: Drupal\domain_config_ui\DomainConfigUIManager
+    arguments: ['@config.storage', '@domain.loader', '@language_manager']
```

Comment:
That's really tricky place, d.o core has a lot of "config override bleeding" issues fixed, also working on  domain_lang module it became clear that priority of loading overrides highly coupled to language manager initialization.
Also some overrides can be done via settings.php and somehow that should affect in UI